*** ding2.info	2012-04-25 11:59:23.000000000 +0200
--- ding3.info	2012-04-25 12:02:14.000000000 +0200
***************
*** 23,26 ****
--- 23,32 ----
  dependencies[] = ding_library_menu
  dependencies[] = ding_news
  
+ ; oc features
+ dependencies[] = fe_block
+ dependencies[] = oc_blocks
+ dependencies[] = views_ui
+ dependencies[] = ding_oc_omega_frontend
+ dependencies[] = sms_service
+ 
  files[] = ding2.profile
