*** ding2.info	2012-04-25 11:59:23.000000000 +0200
--- ding3.info	2012-04-25 12:02:14.000000000 +0200
***************
*** 23,26 ****
--- 23,32 ----
  dependencies[] = ding_library_menu
  dependencies[] = ding_news
  
+ ; oc features
+ dependencies[] = views_ui
+ dependencies[] = ding_oc_latto_frontend
+ dependencies[] = panels_style_collapsible
+ dependencies[] = opening_hours
+ dependencies[] = menu_attributes
+ dependencies[] = diff
+ dependencies[] = module_filter
+ dependencies[] = ting_recommendation_panes
+ dependencies[] = sms_service
+ dependencies[] = advanced_help
+ dependencies[] = alma
+ dependencies[] = git_deploy
+ dependencies[] = webform
  files[] = ding2.profile
